src/transforms/route.rs

```diff
@@ -13,7 +13,6 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize, Debug, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct LaneConfig {
-    #[serde(flatten)]
```

Comment:
Does the absence of the `flatten` affect how input configurations are handled?

Reply:
No. Because `LaneConfigs` are not deserialized. They are created by the `RouteConfig` (line 100). From what I can gather anything that implements `TransformConfig` has to be Deserializable, otherwise I would remove the derive implementation from the struct. I suspect the `flatten` here is just legacy. Everything seems to work fine without it.